services/uploads/serverless.yml

```diff
@@ -219,91 +217,6 @@ resources:
             - ServerSideEncryptionByDefault:
                 SSEAlgorithm: AES256
         AccessControl: Private
-    BucketAVScanRole:
```

Comment:
These two roles are deleted because they're unused, and have been as long as they've been in this file. The permissions are defined above under provider.statements